
=================================== ERRORS ====================================
_ ERROR collecting universal_recon/tests/targeted/test_social_link_parser.py __
import file mismatch:
imported module 'test_social_link_parser' has this __file__ attribute:
  C:\Code\bar-directory-recon\universal_recon\tests\plugins\test_social_link_parser.py
which is not the same as the test file we want to collect:
  C:\Code\bar-directory-recon\universal_recon\tests\targeted\test_social_link_parser.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
=========================== short test summary info ===========================
ERROR universal_recon/tests/targeted/test_social_link_parser.py
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!
